tests/test_statistics_of_labelled_pixels.py

```diff
@@ -218,6 +218,28 @@ def test_statistics_of_labelled_pixels_3d():
 
     assert (np.array_equal(a, b))
 
+def test_standard_deviation():
+    import numpy as np
 
+    image = np.asarray([
+        [1, 2, 3],
+        [4, 5, 6],
+        [7, 8, 9]
+    ])
+
+    labels = np.asarray([
+        [1, 1, 1],
+        [1, 1, 1],
+        [1, 1, 1]
+    ])
+
+    print("numpy standard deviation", np.std(image))
+    print("numpy variance", np.var(image))
+
+    stats = cle.statistics_of_labelled_pixels(image, labels)
+
+    print("std", stats[0].standard_deviation_intensity)
+
+    assert np.abs(stats[0].standard_deviation_intensity - 2.58) < 0.1
```

Comment:
You can do `from numpy import testing` then `testing.assert_almost_equal(stats[0].standard_deviation_intensity, 2.58, decimal=1)`

Reply:
(which has nice features like telling you the value and deviation when the test fails)

Reply:
Awesome! I continure learning, which is great :-)